data/19.0.0.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<release xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../schema.xsd" version="18.0.0">
+  <changelog href="https://nextcloud.com/changelog/#19-0-0"/>
+  <whatsNew lang="en">
+    <regular>
+      <item>Talk 9 integrates document viewing/editing and a grid view</item>
+      <item>Deck integration in Calendar - easily track your deadlines!</item>
+      <item>Assign groups to guest accounts you create</item>
```

Comment:
admin only

Reply:
aaah that should go to admin then somehow. Let me edit it.